Makefile

```diff
@@ -135,7 +136,11 @@ GO_TARGET_PHONY :=
 define COMPILE_GO_TEMPLATE
 
 $(1): SHELL:=$(RUN_IN_BUILD_SHELL)
+ifeq ($(IN_DOCKER),1)
```

Comment:
`IN_DOCKER=1` represent that build in docker. So we should check if IN_DOCKER equal to 0 here.